instrumentation/opentelemetry-instrumentation-tornado/tests/test_instrumentation.py

```diff
@@ -771,3 +774,52 @@ def test_custom_response_headers_not_added_in_internal_span(self):
         self.assertEqual(tornado_span.kind, trace.SpanKind.INTERNAL)
         for key, _ in not_expected.items():
             self.assertNotIn(key, tornado_span.attributes)
+
+
+class ExpectNoLog(logging.Filter):
```

Comment:
This is the inverse of the `ExpectLog` context manager from Tornado: https://github.com/tornadoweb/tornado/blob/f5df43f26bb4d00759176f7cbec8bdce69f2f4f4/tornado/testing.py#L655